AWS.pidx

```diff
@@ -2,19 +2,23 @@
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_Over-the-air_Update" version="4.0.1"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="backoffAlgorithm" version="4.0.0"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreHTTP" version="4.0.0"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreJSON" version="4.0.0"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreMQTT" version="4.0.0"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreMQTT_Agent" version="4.0.0"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="corePKCS11" version="4.0.1"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="FreeRTOS_LTS" version="2021.0.1"/>
-    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="FreeRTOS-Plus-TCP" version="4.0.1"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_Device_Defender" version="4.1.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_Device_Shadow" version="5.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_Fleet_Provisioning" version="1.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_Jobs" version="4.1.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_Over-the-air_Update" version="5.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="AWS_IoT_SigV4" version="1.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="backoffAlgorithm" version="4.1.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreHTTP" version="5.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreJSON" version="4.1.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreMQTT" version="5.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreMQTT_Agent" version="5.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="corePKCS11" version="5.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="coreSNTP" version="1.0.0"/>
+    <pdsc url="https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/" vendor="AWS" name="FreeRTOS_Cellular_Interface" version="1.0.0"/>
```

Comment:
can we standardizse the naming. we're using "-" as the separator in most of the library names (e.g. FreeRTOS-Plus-TCP) but for cellular it is underscore "FreeRTOS_Cellular_Interface". 

Same for the other libraries like coreMQTT_Agent, and OTA.

Reply:
This CAN be done, but you'd have to deprecate the old names - its actually a pretty big hassle... the names were kept from the original beta release.